### PR TITLE
Use a new logo for template

### LINKED ISSUE #111: Use new logo
https://www.openshift.com/about/logos.html

### PR DESCRIPTION
**Pull request by tnir**
- Use `Logotype_RH_OpenShift_wLogo_RGB_Black.svg`
    - obtained with `curl -sSL https://www.openshift.com/images/logos/openshift/Logotype_RH_OpenShift_wLogo_RGB_Black.svg | base64`
- Change aspect
- Add 'Build on' literal out of SVG logo.

Closes #111

### CODE REVIEW

**Comment by tnir** [comment]
Confirmed on Firefox 58 on macOS 10.13 like screenshot below:

oc-django-ex-new-logo

File: welcome/templates/welcome/index.html [hunk 1]
```diff
@@ -311,6 +311,7 @@
         </div>
 
         <footer>
+          <div>Build on</div>
```
Comment (bparees) [inline welcome/templates/welcome/index.html]: s/Build/Built/

Comment (tnir) [reply welcome/templates/welcome/index.html]: Thanks. I'm fixing.

**Comment by tnir** [comment]
@bparees Just updated the commit with the fix.

**Comment by bparees** [comment]
thanks @tnir ! (and for the other updates)

**Pull request merged**